src/lib/containers/UserCardSmall.tsx

```diff
@@ -75,6 +120,7 @@ export const UserCardSmall: React.FunctionComponent<UserCardSmallProps> = ({
       >
         {avatar}
         {`@${userProfile.userName}`}
+        {showAccountLevelIcon && <span className={"account-level-icon"}>{icon}</span>}
```

Comment:
Add account level icon in small user card but only show it when `showAccountLevelIcon` is true.